cli.go

```diff
@@ -0,0 +1,147 @@
+// Copyright © 2019, Oracle and/or its affiliates.
+package ociauth
+
+import (
+	"fmt"
+	"strings"
+
+	"github.com/hashicorp/vault/api"
+
+	"github.com/oracle/oci-go-sdk/common"
+	"github.com/oracle/oci-go-sdk/common/auth"
+	"net/http"
+	"net/url"
+)
+
+type CLIHandler struct{}
+
+func (h *CLIHandler) Help() string {
+	help := `
+Usage: vault login -method=oci authType=ApiKey 
+       vault login -method=oci authType=InstancePrincipal 
+
+  The OCI auth method allows users to authenticate with OCI
+  credentials. The OCI credentials may be specified in a number of ways,
+  listed below:
+
+    1. API Key
+
+    2. Instance Principal
+
+  Authenticate using API key:
+
+		First create a configuration file as explained in https://docs.us-phoenix-1.oraclecloud.com/Content/API/Concepts/sdkconfig.htm
+		Then login using the following command:
+
+		$ vault login -method=oci authType=ApiKey role=<RoleName>
```

Comment:
Use `auth_type` instead for the parameter name. camelCase isn't used for CLI parameters, and `auth_type` has precedent in other backends.

Reply:
Also perhaps: ApiKey -> apikey, InstancePrincipal-> instance

Reply:
done